deploy/utils.sh

```diff
@@ -388,3 +389,68 @@ verify_secret_value_in_pod() {
   $cli_with_timeout "exec -n $APP_NAMESPACE_NAME ${pod_name} -- \
     printenv | grep $environment_variable_name | cut -d '=' -f 2- | grep $expected_value"
 }
+
+get_app_logs_container() {
+    echo "Get logs from the Secrets Provider container"
+    set_namespace "$APP_NAMESPACE_NAME"
+    helm=$(helm ls -aq)
+    $cli_without_timeout get pods
+
+    if [[ -z "$helm" ]]; then
+      pod_name=$($cli_without_timeout get pods --selector app=test-env --no-headers  | awk '{print $1}' )
+      echo "pod_name="$pod_name
+
+      if [[ $pod_name != "" ]]; then
+        $cli_without_timeout logs $pod_name -c cyberark-secrets-provider-for-k8s > "output/$SUMMON_ENV-secrets-provider-logs.txt"
+      fi
+    else
+      pod_name=$($cli_without_timeout get pods --selector app=test-helm --no-headers | awk '{print $1}' )
+      echo "pod_name="$pod_name
+
+      if [[ $pod_name != "" ]]; then
+         $cli_without_timeout logs $pod_name  > "output/$SUMMON_ENV-secrets-provider-logs-with-helm.txt"
+      fi
+    fi
+}
+
+get_conjur_logs_container() {
+    echo "Get logs from the DAP Follower/Conjur container(s)"
+    set_namespace "$CONJUR_NAMESPACE_NAME"
+    selector="role=follower"
+
+    if [ "$CONJUR_DEPLOYMENT" = "oss" ]; then
+      selector="app=conjur-oss"
+    fi
+
+    $cli_without_timeout get pods
+    $cli_with_timeout get pods --selector=$selector --no-headers
+    pod_list=$($cli_without_timeout get pods --selector=$selector --no-headers | awk '{ print $1 }')
+
+    if [[ -z "$pod_list" ]]; then
+      echo "Pod doesn't exist. DAP Follower/Conjur logs were unable to be retrieved "
+    else
+      if [ "$CONJUR_DEPLOYMENT" = "oss" ]; then
+
+        #support in multiple pods on oss
+        for pod in $pod_list
```

Comment:
the format should be `for x in y; do`

Reply:
Its not a mandatory (you can take a look [here](https://www.cyberciti.biz/faq/bash-for-loop/) ) In our code i saw we use both so i will leave it that way because i think its more clear to read the part between do...done.